adafruit_rgb_display/ili9341.py

```
@@ -1,4 +1,8 @@
-import ustruct
+try:
+    import struct
+except:
```

I just realized these should be `except ImportError:` so that they don't hide other errors.

---

ok